content/knowledge_base/consensus/avalanche/specification.md

```diff
@@ -0,0 +1,45 @@
+# Avalanche
+
+An internet-scale electronic payment system, evaluated in a large scale deployment, constructed with Snow family of algorithms. Avalance's tested confirmation latency goes down to 1.35 sec. System's bottleneck is transaction verification.
+TODO: elaborate more on the introduction
+
+
+## Characteristics
+
+Avalanche network operates with multiple single-decree instances (nodes), meaning that each instance can make a decision on only one value. Nodes use Snowball algorithm for the process of making a decision. (TODO: add a reference) In the case of a payment system and Avalanche, the Snowball logic used to determine a value is a process of deciding on a transaction being valid or not. 
+
+The protocol maintains the set of all known transactions using a dynamic, append-only Directed Acyclic Graph (DAG) structure. (TODO: add reference)
+There are two main positives from this:
+ * DAG helps streamlining the path on which there are no conflicting transactions - a single vote on a DAG vertex implicitly votes for all transactions on the path to the genesis vertex
+ * using the same principle as BTC, the DAG tree renders past decisions on the path of transactions as more and more difficult to undo without approval of the correct nodes
+Genesis block concept is implemented with the DAG having a single sink - "genesis vertex".
```

Comment:
```suggestion
The protocol maintains the set of all known transactions using a dynamic, append-only Directed Acyclic Graph (DAG) structure.

There are two main positives from this:

 * DAG helps streamlining the path on which there are no conflicting transactions — a single vote on a DAG vertex implicitly votes for all transactions on the path to the genesis vertex
 * using the same principle as BTC, the DAG tree renders past decisions on the path of transactions as more and more difficult to undo without approval of the correct nodes
 
Genesis block concept is implemented with the DAG having a single sink - "genesis vertex".
```

Also I'm not sure how but I feel like we should rephrase the last sentence, WDYT?

Reply:
Updated the description, IMO is more clear now https://github.com/optakt/optakt.github.io/pull/13/commits/0a1cb902f3417c3bc20f9197df71d3d65a1b6421#diff-69f45991c0732c6f5ff297a5a86ffa91662d50369fc8057b3875361f5b80a2fcR21-R22
WDYT?